airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -447,7 +476,20 @@ def generate_id_hashing_model(self, from_table: str, column_names: Dict[str, Tup
         return sql
 
     def safe_cast_to_strings(self, column_names: Dict[str, Tuple[str, str]]) -> List[str]:
-        return [StreamProcessor.safe_cast_to_string(self.properties[field], column_names[field][1]) for field in column_names]
+        columns_name_safety = [
+            StreamProcessor.safe_cast_to_string(self.properties[field], column_names[field][1]) for field in column_names
+        ]
+        if self.destination_type == DestinationType.ORACLE:
+            oracle_cols_safe = []
+            for col in columns_name_safety:
+                if "(" in col:
+                    oracle_cols_safe.append("{{" + col + "}}")
+                else:
+                    oracle_cols_safe.append(col)
+
+            return oracle_cols_safe
```

Comment:
Could we get some explanations why this is needed?

Reply:
Sure!

Reply:
# this inject jinja macro in Oracle queries
        # for Oracle you need to be explicit when calling jinja macros in queries
        # array_to_string(field), QUOTE(field) is a dbt macro but in oracle will throw an error
        # so you need to specify them as {{ array_to_string(field) }}.


WDYT?